README.md

```diff
@@ -51,11 +51,13 @@ $ jiraprinter --help
 
   Options:
 
-    -h, --help               output usage information
-    -V, --version            output the version number
-    -u, --user [username]    The JIRA username ($JIRA_USER)
-    --password [password]    The JIRA password ($JIRA_PASS)
-    -h, --host [host]        The JIRA hostname ($JIRA_HOST)
+    -h, --help                                  output usage information
+    -V, --version                               output the version number
+    -u, --user [username]                       The JIRA username ($JIRA_USER)
+    --password [password]                       The JIRA password ($JIRA_PASS)
+    -h, --host [host]                           The JIRA hostname ($JIRA_HOST)
+    -pqr --printQR [printQR]                    Whether to include an issue link QR code in the print out ($PRINT_QR)
```

Comment:
It is only false right now because I had not changed it back from my testing today ;) I want to maintain as much of the existing default options as possible!